vmdb/lib/vmdb/config.rb

```diff
@@ -162,10 +162,10 @@ def normalize_time(input)
     def config_mtime_from_db
       #log_header = "MIQ(Config.config_mtime_from_db) [#{@name}]"
       server = MiqServer.my_server(true)
-      return if server.nil?
+      return Time.at(0) if server.nil?
```

Comment:
nice find !!

Should we be doing something else when checking values returned from this method  ?  i.e. line 314/315, that message is no longer getting logged.  do we even need it. Thanks.